test/schemas/v0/tests.schema.json

```diff
@@ -347,6 +342,23 @@
               },
               "value": {}
             }
+          },
+          {
+            "description": "Fallback part.",
```

Comment:
Can we be more verbose with the descriptions? They're here as documentation, after all...

Reply:
This is literally a fallback part, i.e. the result of formatting a value that experienes fallback. I don't really know what else to say about it?